utils/metrics/metrics_util.go

```diff
@@ -316,3 +318,18 @@ func IncrConsumerEventCount(msgType string) {
 		[]metrics.Label{telemetry.NewLabel("msg_type", msgType)},
 	)
 }
+
+// Gauge for gas price paid for transactions
+// Metric Name:
+//
+// sei_evm_effective_gas_price
+func GaugeEvmEffectiveGasPrice(gasPrice *big.Int, blockNum uint64, txHash common.Hash) {
```

Comment:
I'm thinking we would want this to be an increment, so we can aggregate on it (e.g., avg gas price).  It's possible that gauge has some kind of capability like that but I thought it was more of a replace concept  (set value to X).  I could be wrong on that.